cfn-resources/federated-database-instance/test/cfn-test-delete-inputs.sh

```diff
@@ -21,3 +21,13 @@ else
 	(echo "Failed cleaning project:$projectId" && exit 1)
 fi
 
+keyRegion=$AWS_DEFAULT_REGION
+if [ -z "$keyRegion" ]; then
+	keyRegion=$(aws configure get region)
+fi
+
+echo -e "--------------------------------delete aws bucket starts ----------------------------\n"
```

Comment:
knit: i would probably removed both echos if you're already sure that it's working